src/flask_api.py

```diff
@@ -0,0 +1,187 @@
+from flask import Flask, request
+import json
+import numpy as np
+import gdown
+
+from fastai.vision import load_learner
+
+from . import predict as pred
+from . import transform_data as td
+
+# import predict as pred
+# import transform_data as td
+
+import base64
+import io
+from PIL import Image
+from pathlib import Path
+
+app = Flask(__name__)
+host = 'localhost'
+port = '5000'
+
+# load model
+try:
+    model_path = Path(__file__).parents[1]
+    learn = load_learner(model_path, 'stage-2_bs16.pkl')
+except IOError:
+    url = 'https://drive.google.com/uc?id=1ClEUOwFhIOMNxvedps8WQN8Koy6NT5eI'
```

Comment:
You may wanna put this URL in your configs, as the location might end up changing later.